recipes/leftbar/group_row/group_row.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <dt-recipe-general-row
+    :description="names"
+    :unread-count="unreadCount"
+    :selected="selected"
+    v-on="$listeners"
+  >
+    <template #left>
+      <dt-avatar
+        :initials="avatarInitials"
+        :group="groupCount"
+      >
+        <img
+          data-qa="dt-avatar-image"
+          :src="avatarSrc"
+          :alt="avatarInitials"
+        >
+      </dt-avatar>
+    </template>
+  </dt-recipe-general-row>
+</template>
+
+<script>
+import { DtRecipeGeneralRow } from '@/recipes/leftbar/general_row';
+import DtAvatar from '@/components/avatar/avatar.vue';
+
+export default {
+  name: 'DtRecipeGroupRow',
+
+  components: {
+    DtAvatar,
+    DtRecipeGeneralRow,
+  },
+
+  props: {
+    /**
+     * Optional avatar image url.
+     * if provided, it's also required to provide a value in the `avatarInitials` prop to use
+     * in the alt attribute of the avatar.
+     */
+    avatarSrc: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Initials to display on the avatar if avatarSrc is not provided or
+     * alt attr if avatarSrc is provided.
+     */
+    avatarInitials: {
+      type: String,
+      default: '',
+    },
```

Comment:
Perhaps more a bug of Avatar's Group variant, but if the Group Avatar doesn't have an image the initials are not displayed at that size (Avatar size XS). 

<img width="527" alt="image" src="https://user-images.githubusercontent.com/1165933/221310481-1fff3774-16b5-4d58-a7ab-5085f4ec6e8a.png">

Should be like so...

<img width="32" alt="image" src="https://user-images.githubusercontent.com/1165933/221310743-1c5172a8-b8eb-4a9c-a6ab-7af325213b61.png">
 
...which I get from this combo https://vue.dialpad.design/deploy-previews/pr-778/?path=/story/components-avatar--initials&args=size:xs;group:2

Reply:
Yeah it was a bug in the Avatar group since with variant it changes the size to `xs` with CSS but the component size remains with the size passed with the `size` prop. Updated and fixed this.
Can you validate that [the logic here](https://github.com/dialpad/dialtone-vue/pull/778/commits/09abfa9bcaf1b5a89c60877486d1d7f3cb9a5c7c#diff-90b30c664802d78094183e71b1acbb8385122479a20e6052588cd5a6d422dff0R201) is okay?

Reply:
Logic looks sound to me. I played around with a few combinations of `size` and `group` to try to break it, e.g. https://vue.dialpad.design/deploy-previews/pr-778/?path=/story/components-avatar--initials&args=size:lg;group:2

Keep playing with those values to see if you can you break it.